### ISSUE TITLE
After activating 30-days trial I got the "Account Verification Message" email and something is wrong with it

### ISSUE THREAD

**Issue by limonte** [FlowCrypt/flowcrypt-browser#1901]
![image](https://user-images.githubusercontent.com/6059356/62851919-76093a80-bcf0-11e9-8d1a-37df4f31cdcc.png)

```
Could not complete: Error: Api error response when POST-ing 
https://flowcrypt.com/api/account/login string: account,uuid,token
```

---

Also, I'm not sure why did I get this email and how is "verification" related to activating of 30-days free trial period.

**Comment by tomholub** [comment]
Very good (or very bad, depending how you look at it) :+1:

**Comment by tomholub** [comment]
I have to look at this

**Comment by tomholub** [comment]
Basically, when signing up for a trial, the extension will want to verify the account owner's email (just like when signing up for any other website with email address). Like an account login with our backend. It will send a verification email. Instead of having user click on it, FlowCrypt extension will be looking for the email, grab the verification token from it and POST it back to backend for verification.

Which failed in your case.

**Comment by tomholub** [comment]
This will be fixed with #2011
